packages/notification-controller/src/NotificationController.test.ts

```diff
@@ -51,16 +51,37 @@ describe('NotificationController', () => {
     expect(
       // TODO: Either fix this lint violation or explain why it's necessary to ignore.
       // eslint-disable-next-line @typescript-eslint/await-thenable
-      await unrestricted.call('NotificationController:show', origin, message),
+      await unrestricted.call('NotificationController:show', origin, {
+        message,
+      }),
+    ).toBeUndefined();
+
+    expect(
+      // TODO: Either fix this lint violation or explain why it's necessary to ignore.
+      // eslint-disable-next-line @typescript-eslint/await-thenable
+      await unrestricted.call('NotificationController:show', origin, {
+        message,
+        title: 'title',
+        detailedView: '1',
+      }),
```

Comment:
Nit: can we remove the TODO and async here too
```suggestion
       unrestricted.call('NotificationController:show', origin, {
        message,
        title: 'title',
        detailedView: '1',
      }),
```